Add kubeflow on aws docs

Jeffwan opened the pull request:
AWS recently add some aws specific features to kubeflow and we also like to have these docs to help users understand features. 

Address doc issue.
https://github.com/kubeflow/website/issues/621

We have a static server here. Links in the main page seems doesn't work because of hugo config issue, I use a nginx to proxy traffic. Please click link on left sidebar to enter page. 
http://54.213.19.198/docs/aws/

Code changes for aws:
https://github.com/kubeflow/kubeflow/pull/2823
https://github.com/kubeflow/kubeflow/pull/2983

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubeflow/website/657)
<!-- Reviewable:end -->

Jeffwan commented:
/cc @sarahmaddox

Jeffwan commented:
Does the bot automatically recheck CLA? @nrdlngr already signed CLA for himself.

nrdlngr commented:
Approved.

sarahmaddox commented:
/assign

sarahmaddox commented:
@Jeffwan We have Netlify running, which means you get a nice preview when you submit a docs PR.

The preview for this PR is here:
https://deploy-preview-657--competent-brattain-de2d6d.netlify.com/docs/aws/

And here:
https://deploy-preview-657--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-aws/

sarahmaddox commented on content/docs/aws/authentication.md:
```diff
@@ -0,0 +1,68 @@
+## Traffic Flow
+External Traffic → [ Ingress → Istio ingress gateway → ambassador ]
+
+When you generate and apply kubernetes resources, an ingress is created to manage external traffic to Kubernetes services. The AWS ALB Ingress Controller will provision an Application Load balancer for that ingress. By default, TLS and authentication are not enabled at creation time.
```
Please expand or explaing "ALB" on first usage.

sarahmaddox commented on content/docs/aws/deploy/install-kubeflow.md:
````diff
@@ -0,0 +1,151 @@
+
+        ```shell
+        kubectl port-forward -n kubeflow `kubectl get pods -n kubeflow --selector=service=ambassador -o jsonpath='{.items[0].metadata.name}'` 8080:80
+        ```
````
My above comment is probably useful on other pages in this PR too.

sarahmaddox commented on content/docs/aws/features.md:
```diff
@@ -0,0 +1,16 @@
+
+Running Kubeflow on Amazon EKS brings the following optional and configurable features:
+
+* You can manage your Amazon EKS cluster provisioning with **eksctl** and easily choose between multiple compute and GPU worker node configurations.
```
Suggestion: Add a link for each of the features on this page, so people who don't know what they are can easily find more info.

sarahmaddox commented on content/docs/aws/deploy/existing-cluster.md:
```diff
@@ -0,0 +1,47 @@
++++
+title = "Init cluster setup for existing cluster"
```
s/Init/Initial

sarahmaddox commented:
@googlebot please re-check CLAs. All contributors have signed and approved this PR.

sarahmaddox commented:
/ok-to-test

Jeffwan commented on content/docs/aws/deploy/install-kubeflow.md:
```diff
@@ -0,0 +1,151 @@
++++
+
+This guide describes how to use the `kfctl.sh` script to
+deploy Kubeflow on AWS.
```
Good point. I will address it in the doc.

Jeffwan commented:
@sarahmaddox  Thanks a lot for your review! I addressed all the feedbacks and update docs. Please have another look.

Another thing we can not clear is CLA issue.  @nrdlngr signed CLA and how could he verify author consent? We are not sure why command bot accepts and he left `Approved` here.

` CLAs are signed, but unable to verify author consent`

jlewi commented:
To fix the CLA check; you need to rebase and change the commit authors so its a single author who matches PR author.

If you rebase and squash commits that will fix it.

Jeffwan commented:
@sarahmaddox Double check bot message and notice `cla/google` won't change to `Yes`. Approval can manually update label or move forward directly. Since @nrdlngr already agree to contribute, I think we are good on CLA parts.

sarahmaddox commented:
Thanks! Agreed we're good to go on the CLA. I'll approve this and see if the merge happens automatically, as we don't have the CLA label in this repo. If that doesn't work, I'll try merging myself.

/lgtm
/approve
